static/js/utils/disaster_event_map_utils.tsx

```diff
@@ -240,18 +256,26 @@ function fetchEventPoints(
         const severity = {};
         if (severityFilter && severityFilter.prop in eventData.propVals) {
           const severityVals = eventData.propVals[severityFilter.prop].vals;
-          if (!_.isEmpty(severityVals)) {
-            // Get value by taking the first severity val, trimming the unit
-            // from the string, and converting it into a number.
-            const unit = severityFilter.unit || "";
-            console.assert(
-              severityVals[0].length > unit.length,
-              "severity values do not contain unit, please check filter config, %o",
-              reqParams
-            );
-            const val = Number(severityVals[0].substring(unit.length));
-            if (!isNaN(val)) {
-              severity[severityFilter.prop] = val;
+          const val = parseEventPropVal(
+            severityVals,
+            severityFilter.unit || ""
```

Comment:
should we also pass reqParams in this call to parseEventPropVal?

Reply:
yes! done.